### ISSUE TITLE
Indentation breaks when having protocol conformances on multiple lines

### ISSUE THREAD

**Issue by rastersize** [nicklockwood/SwiftFormat#561]
Heya 🙂 Looks like there’s some kinda bug in the indentation. Given the following code (we use this to split the line when the list of protocols would extend past our line length):

```swift
class Foo: Bar,
    SomeProtocol {
    let test: String
    init() {}
}
```

This will be corrected as:

```swift
class Foo: Bar,
    SomeProtocol {
        let test: String
        init() {}
    }
```

This is with SwiftFormat 0.44.2. It was working fine in 0.40.14 which is the version we where at before. We tried bumping in hope that it would solve another issue*.

<details><summary>.swiftformat</summary>

```shell
# Rule configurations
--allman false
--binarygrouping 4,8
--commas always
--comments indent
--decimalgrouping 3,5
--elseposition same-line
--empty void
--exponentcase lowercase
--exponentgrouping disabled
--fractiongrouping enabled
--header strip
--hexgrouping none
--hexliteralcase lowercase
--ifdef noindent
--indent 4
--indentcase false
--linebreaks lf
--octalgrouping 4,8
--operatorfunc spaced
--patternlet inline
--nospaceoperators ...,..<,..>
--self remove
--semicolons inline
--stripunusedargs closure-only
--trimwhitespace always
--wraparguments beforefirst
--wrapcollections beforefirst
--closingparen balanced

# Disabled rules
--disable numberFormatting

# Enabled rules

# Tool options
--symlinks ignore
```

</details>

---

<details><summary>The other issue</summary>
Code like this gets formatted oddly:

```swift
func foo(foo: Foo) -> () -> Void {
    {
        if foo.bar {
            doSomething()
        } else {
            meh()
        }
    }
}
```

It ends up getting formatted as:

```swift
func foo(foo: Foo) -> () -> Void { {
    if foo.bar {
        doSomething()
    } else {
        meh()
    }
} }
```

</details>

**Comment by nicklockwood** [comment]
@rastersize thanks for reporting this, I'll look into it.

**Comment by nicklockwood** [comment]
@rastersize fixed in 0.44.5
